samples/net/download/boards/nrf9161dk_nrf9161_ns.conf

```diff
@@ -0,0 +1,32 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Configuration file for nRF9160 DK
+# This file is merged with prj.conf in the application folder, and options
+# set here will take precedence if they are present in both files.
+
+# General
+CONFIG_HW_STACK_PROTECTION=y
+CONFIG_PICOLIBC=y
```

Comment:
These are both defaults, no need to specify.

Reply:
The HW_STACK_PROTECTION is default, though the PICOLIBC is set no n when I disable it.

Reply:
What do you mean by "disable it"? You should remove this line to get the default value, if you specify `CONFIG_PICOLIBC=n` then you are explicitly setting it to `n`

Reply:
I didn't mean disable, just a spelling error, I removed it.